.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
- [ ] Kindly ensure you update your linters config by using the required ones on this project. Right now, you are still using the **HTML-CSS** linters instead of **[HTML-CSS-JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js)** ones. I suggest you fix that as you should also check for **Javascript** errors. **[Here](https://github.com/microverseinc/linters-config/tree/master/html-css-js)** are are all information you need about the configuration.